ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -192,6 +231,65 @@ tests:
           cpu: 500m
           memory: 1Gi
     workflow: ipi-gcp
+- as: cluster-api-provider-kubevirt-with-hypershift
+  steps:
+    cluster_profile: gcp
+    test:
+    - as: deploy-cnv
+      commands: ./hack/ci/install-cnv.sh
+      from: hypershift-cli-e2e
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    - as: deploy-hypershift
+      commands: |-
+        set -x
+        hypershift install
+        oc patch deployment -n hypershift operator --type=json -p '[{"op": "add", "path": "/spec/template/spec/containers/0/env/-", "value": {"name": "IMAGE_KUBEVIRT_CAPI_PROVIDER", "value": "'"${CAPK_IMAGE}"'"}}]'
+        # make sure the patch worked
+        oc get deployment -n hypershift operator -o yaml
+        oc wait deployment -n hypershift operator --for=condition=Available --timeout=5m
```

Comment:
I'm not sure this guarantees that the new operator has rolled out. We'd have to verify status.UpdatedReplicas == spec.Replicas i think.